### ISSUE TITLE
Re-enable and Monitor `YAMLSchemaValidation` inspection

### ISSUE THREAD

**Issue by MANISH-K-07** [checkstyle/checkstyle#14806]
Follow-up of https://github.com/checkstyle/checkstyle/pull/14696#pullrequestreview-2006660651

[inspection-results/YAMLSchemaValidation.xml
](https://output.circle-artifacts.com/output/job/ab162cce-e22e-4921-9f14-8dc473901855/artifacts/0/home/circleci/project/target/inspection-results/YAMLSchemaValidation.xml)We had stable failure of inspection validation `YAMLSchemaValidation`
https://app.circleci.com/pipelines/github/checkstyle/checkstyle/23624/workflows/e04a197a-8172-4262-9fa3-de02579854c4/jobs/506740/artifacts

```
<problems is_local_tool="true">
<problem>
<file>file://$PROJECT_DIR$/.github/workflows/site.yml</file>
<line>14</line>
<module>project</module>
<entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/.github/workflows/site.yml"/>
<problem_class id="YAMLSchemaValidation" severity="WARNING" 
      attribute_key="WARNING_ATTRIBUTES">Validation by JSON Schema</problem_class>
<description>Schema validation: Property 'name' is not allowed</description>
<highlighted_element>name: "Site"</highlighted_element>
<language>yaml</language>
<offset>0</offset>
<length>12</length>
</problem>
```

This problem is a false positive. Facing instability with this inspection, it had been disabled at https://github.com/checkstyle/checkstyle/commit/65314ea6904f8a09482d6d1f673dc4fd715f0c77

We have migrated IDEA to highest true scopes release `v2022.3.3` at #14696 

~~Hoping that the migration has fixed this problem~~ (https://github.com/checkstyle/checkstyle/issues/14806#issuecomment-2063439340), 
we need to try and re-enable this inspection.

https://github.com/checkstyle/checkstyle/blob/0707dd5ac38b2f0fd4af9941070ea2615c17f335/config/intellij-idea-inspections.xml#L4109-L4111

**Comment by MANISH-K-07** [comment]
@romani , @nrmancuso , You can track re-runs of inspections for success/failure at https://github.com/checkstyle/checkstyle/pull/14808#issuecomment-2063268642

I will keep restarting CI and updating that comment with links until you are satisfied for merge ;)

**Comment by MANISH-K-07** [comment]
Unfortunately, the problem still persists despite migration of IDEA to v2022.3.3 :(
https://output.circle-artifacts.com/output/job/4946fcff-c418-4172-b566-5e343cb6d7b7/artifacts/0/home/circleci/project/target/inspection-results/YAMLSchemaValidation.xml

@romani , what do you suggest we do? Report this to jetBrains?

**Comment by nrmancuso** [comment]
>what do you suggest we do? Report this to jetBrains?

Unless you can show that it still happens on the latest version, I am not sure if they will give it much attention.

**Comment by MANISH-K-07** [comment]
> Unless you can show that it still happens on the latest version, I am not sure if they will give it much attention.

Honestly, there's been no activity at scopes issue itself despite the fact that it impacts latest releases. Please see [IJPL-1068](https://youtrack.jetbrains.com/issue/IJPL-1068)
Me being used to everything getting done so efficiently here at CS, it seems a bit strange that they take so long even after issue is assigned 😅 

> I have commented elsewhere previously that it may be a network/server problem rather than an actual "bug" in the inspections engine. The yaml schema is downloaded on each execution of inspections.

@nrmancuso , so how long can we keep it disabled then? Is there any alternative?

**Comment by romani** [comment]
>Unfortunately, the problem still persists despite migration of IDEA to v2022.3.3 :(

Please close PR and let's wait for our migration to latest Idea.

**Comment by romani** [comment]
Activated
